src/sentry/static/sentry/app/components/forms/state.tsx

```diff
@@ -1,8 +1,10 @@
-export default {
-  HOVER: 'Hover',
-  DISABLED: 'Disabled',
-  LOADING: 'Loading',
-  READY: 'Ready',
-  SAVING: 'Saving',
-  ERROR: 'Error',
-} as const;
+enum FormState {
+  HOVER = 'Hover',
+  DISABLED = 'Disabled',
+  LOADING = 'Loading',
+  READY = 'Ready',
+  SAVING = 'Saving',
+  ERROR = 'Error',
```

Comment:
```suggestion
  HOVER = 'hover',
  DISABLED = 'disabled',
  LOADING = 'loading',
  READY = 'ready',
  SAVING = 'saving',
  ERROR = 'error',
```

Reply:
we usually write them lowercase  😉

Reply:
Yeah I agree. I think I would prefer to avoid changing these though in the case where we might be comparing these somewhere wonky.

We might even just be using these as text strings. I'm not quite sure.